seed/scripts/update-prices.ts

```diff
@@ -0,0 +1,26 @@
+import { BadgerAPI, Currency, Network } from '@badger-dao/sdk';
+import { writeFileSync } from 'fs';
+import { resolve } from 'path';
+
+async function updatePrices() {
```

Comment:
```
npx ts-node seed/scripts/update-prices.ts
```
this will now allow users to update price seeds to not have to wait on cold starts for pricing updates